Doc/library/importlib.metadata.rst

```diff
@@ -167,7 +175,7 @@ Inspect the resolved entry point::
 
 The ``group`` and ``name`` are arbitrary values defined by the package author
 and usually a client will wish to resolve all entry points for a particular
-group.  Read `the setuptools docs
+group. Read `the setuptools docs
```

Comment:
I also prefer single spaces but please don't change this, we prioritise avoiding churn.

https://devguide.python.org/documentation/style-guide/#use-of-whitespace

Reply:
My bad, what I get for being lazy and editing online. Was thinking about Putting the new info there but decided it's better further up where the filtering is showcased. Fixed